.gitignore

```diff
@@ -16,4 +16,7 @@ sub-projects/hub/public/*.json
 sub-projects/hub/public/*.js
 
 # Generated Documentation
-sub-projects/hub/docs/
\ No newline at end of file
+sub-projects/hub/docs/
+package.json
+yarn.lock
+sub-projects/device/utils/__pycache__/compare.cpython-36.pyc
```

Comment:
You might want to make a different category for this. It's technically compiled files, not documentation, right?